src/mgr/MgrStandby.cc

```diff
@@ -160,8 +161,25 @@ void MgrStandby::send_beacon()
   auto addr = available ? active_mgr->get_server_addr() : entity_addr_t();
   dout(10) << "sending beacon as gid " << monc.get_global_id()
 	   << " modules " << modules << dendl;
-
+  
+  const char *host_ip = NULL;
+  char addr_buf[129];
```

Comment:
Yes, this is more elegant.
For INET and INET6 do a different definition, but also do not need "+1"